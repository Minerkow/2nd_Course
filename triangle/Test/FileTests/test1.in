100


 4.32825 1.44383 -0.790067  3.97477 1.1575 0.551572  4.5398 1.66084 0.691077
 2.94666 1.92034 2.31673  2.88211 2.28556 2.71031  3.25904 2.11689 2.57078
 0.23822 -0.454261 1.85722  0.0312358 -0.663097 3.02169  0.123984 0.0846242 1.79038
 5.22717 3.07661 0.182032  4.6976 3.45875 -0.33631  4.59017 3.17505 -0.456289
 3.55702 3.34708 -0.0433232  3.71648 3.38007 0.130049  3.60682 3.39894 0.084339
 0.939769 -0.180646 1.33931  1.03441 -0.121284 1.42936  0.748532 0.0270237 1.43245
 -0.023369 1.59533 3.12075  -0.0332404 1.70695 2.83324  -0.269041 1.70555 3.07072
 1.52615 -0.1895 0.907226  1.26242 -0.055756 0.99677  1.14717 0.0938062 1.80346
 1.34873 0.707657 3.78283  1.16714 0.271971 3.80221  1.14183 0.539722 3.71981
 1.93614 1.70685 3.38352  1.9308 1.70988 3.3803  1.94004 1.69807 3.3625
 3.74852 4.74416 4.91957  3.61205 4.73777 4.87715  3.74908 4.71617 4.88355
 4.64595 2.55588 -0.964845  3.40291 3.31594 -0.969147  3.1763 2.54361 0.347793
 3.18414 2.95476 0.309493  3.56011 2.7592 0.818626  3.54318 2.86728 0.86424
 2.78843 0.718557 4.5791  2.71181 0.748787 4.78684  2.95626 0.709101 4.8342
 1.36148 1.05099 4.81883  1.39153 1.08968 4.70881  1.40565 1.09035 4.70464
 2.06249 4.0877 1.80999  2.00778 4.19198 1.93422  1.94986 4.12929 1.91934
 -0.211191 3.03751 0.426935  -0.469694 2.06043 -0.0554112  -0.158694 1.95519 -0.0687144
 2.75211 -0.0727304 1.41225  2.88128 -0.0105317 1.3547  2.83839 0.113789 1.3145
 3.77225 0.999104 -0.223987  4.15459 0.649806 0.725616  4.22921 1.00432 -0.466359
 0.107117 0.695375 2.67825  -0.425813 0.166622 1.97052  -0.20124 0.899102 1.68151
 3.35444 4.3442 -0.987463  3.49687 5.57437 -0.294162  3.06375 4.16804 -0.965237
 3.17595 3.51016 -0.418913  3.17741 3.50327 -0.433677  3.17476 3.5048 -0.432439
 2.50555 3.44957 4.72817  2.80548 3.84282 4.03277  2.72338 4.29726 4.06305
 2.01807 3.1001 3.72744  1.45996 2.89898 4.94511  0.997503 4.29646 3.95617
 1.29383 -0.67733 2.66946  1.69517 0.248063 1.96143  2.16449 -0.0795658 1.74832
 4.72087 0.178151 1.55971  4.67195 0.371005 1.48343  4.46035 0.308065 2.10927
 4.29613 4.64862 4.41612  4.63457 5.16114 4.62304  4.33347 5.15549 4.81653
 2.958 4.26127 0.538083  3.06461 5.00037 -0.215886  3.5465 4.48388 0.638293
 0.00811473 0.284177 1.78336  -0.475307 0.651963 1.96645  -0.581018 0.89731 1.76343
 0.237845 1.95047 1.05588  0.462518 1.73837 0.725664  0.0281284 1.63465 0.870176
 -0.47004 1.19923 4.23979  0.482061 1.63027 5.44898  -0.359031 0.937001 5.47686
 3.01209 1.82443 -0.41182  2.95394 1.86225 -0.563987  2.94984 1.89344 -0.562653
 2.35145 3.14298 -0.40137  2.50085 3.19067 -0.66612  2.33949 3.22632 -0.59081
 3.94521 1.99258 4.65417  3.41434 1.42559 4.77002  3.79555 1.60095 3.81133
 4.0016 3.07486 2.09457  4.16108 4.10663 1.77687  3.88264 4.12279 3.09968
 4.63155 3.90452 4.62569  4.44052 3.79741 4.2793  4.63004 3.50045 4.41912
 4.44489 -0.196293 -0.223618  4.45734 -0.395609 -0.289375  4.3806 -0.317341 -0.218577
 4.22728 -0.726293 4.74642  4.52785 -0.217343 3.79663  3.83407 -0.096128 3.92177
 -0.163444 0.260425 1.32238  -0.163911 0.328442 1.32571  -0.180968 0.193785 1.48163
 0.198841 2.85051 2.10356  -1.13898 2.6424 2.68057  -0.0370025 1.60096 2.08437
 3.95511 2.18809 0.247359  4.67887 1.78717 0.938925  4.32649 1.5975 0.365267
 1.04863 2.83267 3.58126  1.00589 3.05746 3.23827  1.00158 2.7783 3.55605
 4.40633 1.46241 5.02267  4.20847 1.45911 5.09284  4.147 1.89779 4.66769
 -0.531913 2.09792 2.94257  0.347887 1.47902 3.95973  -0.607701 1.89704 4.3228
 1.53836 2.21567 1.22061  1.21321 2.24977 1.20028  1.40441 1.58327 0.0587602
 1.22663 1.8589 3.20157  1.23935 1.89772 3.21252  1.2366 1.90628 3.21289
 2.137 -0.178698 4.14245  0.979106 -0.226398 3.47224  1.77814 -1.18472 3.42291
 1.77037 -0.424076 4.04198  2.04215 -0.333493 3.69874  2.06154 -0.392528 3.98214
 0.0974384 2.1723 0.0979782  -0.409514 1.74654 -0.249636  0.0231159 0.958914 0.707778
 1.74033 0.305844 2.06711  1.45828 0.952572 2.76141  1.1109 0.435494 2.8272
 1.5893 3.55911 4.14667  0.961256 3.52132 3.72166  1.03854 3.80186 4.00515
 4.55915 -0.277784 1.21921  4.46571 -0.386923 1.5462  4.70181 -0.461464 1.23162
 -0.085849 1.62451 -0.374159  0.350645 1.06774 -0.220305  0.229897 0.884849 -0.155274
 4.35359 0.901629 2.71825  3.96514 0.76623 2.65791  4.13084 1.1439 3.52468
 -0.193701 -0.549446 -0.553867  -0.113623 -1.05315 0.81139  0.163277 -0.406027 -0.662364
 1.66379 2.92693 -0.0641164  1.76764 2.77949 -0.149823  1.75059 2.88867 -0.0907712
 1.27558 0.79459 4.50621  1.27394 0.791648 4.50751  1.27558 0.79452 4.51155
 3.69983 1.39923 1.04886  3.68683 2.36769 1.80623  3.5999 2.27219 0.737133
 -0.0585883 4.06056 4.90517  0.446224 3.81619 4.24701  0.0670601 3.5786 4.11225
 2.25977 0.132467 0.068943  2.22456 0.131104 0.0183864  1.35856 0.244679 0.189079
 -0.30454 1.74671 3.35584  -0.531986 1.0315 4.69313  -0.653589 1.62046 4.80393
 0.214614 2.79081 4.25702  0.0680441 2.85033 3.51265  0.436483 2.83144 3.6574
 2.38895 3.50392 3.94831  2.45025 3.41566 4.24187  2.2292 3.35112 4.32552
 0.822114 0.554777 1.17625  0.851317 -0.0966895 1.1461  -0.158587 -0.23231 1.91111
 0.210788 3.00901 2.70554  -0.162797 3.36943 2.99139  -0.166301 2.81037 1.95075
 2.92493 2.889 -0.303166  3.65531 3.54441 0.418194  3.73598 2.71482 -0.576027
 0.697015 0.72115 3.81791  0.746139 1.23022 5.03432  0.80457 1.62165 4.22349
 0.442634 1.50475 2.62331  0.400876 1.44528 2.6992  0.380446 1.47477 2.71741
 1.34586 2.25094 3.72596  1.35104 2.24732 3.64817  1.37258 2.23184 3.64477
 1.18936 3.2593 0.648256  0.149807 3.26707 0.423507  0.0918065 3.31457 1.97959
 3.97441 0.00206068 0.0216894  4.05176 0.435273 0.354455  4.0895 -0.252549 1.02749
 1.37754 2.18713 0.301928  1.44313 2.2617 0.287907  1.54285 2.09194 0.766834
 0.39017 2.61562 3.88638  0.33504 2.6793 4.36431  0.176673 2.84472 4.26054
 5.4139 2.2097 2.68123  5.16653 2.37176 2.7968  4.55989 1.6907 1.83987
 4.08678 3.2911 3.86548  4.58356 3.56197 4.24242  4.19283 3.67168 4.49198
 -0.408376 -0.0962927 1.05059  -0.111372 0.16806 1.00623  -0.799359 -0.208173 1.44496
 1.18809 3.58925 2.25997  1.17162 3.50169 2.37013  1.28125 3.46573 2.29918
 2.60115 3.10811 0.885073  2.32694 3.31884 0.945479  2.46882 3.00427 1.14776
 3.83106 0.587469 3.03649  3.10663 0.0249798 4.546  3.64078 0.0413838 4.66555
 0.780816 4.23976 1.36298  0.884237 4.98278 1.35766  -0.02771 4.38125 0.582734
 4.2221 0.937938 2.26516  4.28809 1.19375 2.35006  4.07732 0.815712 1.61947
 0.972769 3.39073 4.30687  1.1112 3.50365 4.38384  0.922581 3.33978 4.69156
 2.96877 2.36261 1.7793  2.82966 1.8861 1.67959  3.05762 2.42179 1.68921
 0.625218 -0.852728 2.28142  0.514387 -0.796354 2.29448  0.675071 -0.0871786 2.27302
 3.02739 1.80646 1.45823  3.34086 0.638882 1.80016  2.95814 1.17135 1.17248
 4.35051 -0.506274 1.96106  4.30383 -0.52261 0.916465  4.89692 0.0363865 1.0564
 4.13125 2.29849 -0.108254  4.18419 2.34724 -0.18092  4.19412 2.29411 -0.147006
 2.48725 2.15903 3.35605  3.07693 2.59242 3.68606  3.28908 1.92415 3.09643
 4.1887 1.59908 2.98077  4.04656 1.3219 3.02803  3.97346 1.87753 2.81822
 0.154918 1.53135 4.26896  0.29598 0.620887 4.41474  -0.303258 1.22948 4.42436
 0.0443306 3.12481 2.41939  0.231334 2.78372 1.44757  -0.0122425 3.36902 2.26885
 0.174033 -0.240322 -0.322958  -0.222534 0.643163 0.2812  -0.0296412 0.327608 -0.36408
 4.81492 2.89832 3.71882  4.77219 2.69382 3.57729  4.97737 2.72204 3.62948
 -0.329972 3.88801 -0.195752  -0.911631 4.36583 1.09841  0.148328 3.8104 1.11355
 3.36198 0.515657 0.879574  3.30049 0.533169 1.03392  3.34013 0.5677 1.0298
 2.01137 0.757769 2.10341  2.23542 0.900027 2.15145  2.37628 0.621609 2.30362
 3.75617 3.71856 3.79127  3.89636 3.82098 4.30738  3.82516 3.79794 4.28399
 1.83137 2.66155 4.04825  0.992043 2.04367 2.69457  1.96943 2.06161 2.53449
 3.24482 0.377204 3.97744  2.89115 0.299281 3.89143  3.50822 -0.0205076 3.49138
 3.92243 2.47676 1.40918  4.19948 2.34179 1.60011  3.79421 2.35758 2.11501