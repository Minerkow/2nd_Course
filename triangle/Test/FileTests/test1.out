28 19 61 23 47 65 50 11 72 46 85 51